CHANGELOG.md

```diff
@@ -1,3 +1,350 @@
+- Add Rook Ceph upgrade guide ([#1165](https://github.com/kinvolk/lokomotive/pull/1165)),
+
+#### Others
+
+- Pull control plane images from Quay ([#1226](https://github.com/kinvolk/lokomotive/pull/1226)). To avoid hitting Docker Hub pulling limits.
+- Bootkube now waits for all controlplane charts to converge before exiting, which should make bootstrapping process more stable ([#1085](https://github.com/kinvolk/lokomotive/pull/1085)).
+
+### Upgrading from v0.5.0
+
+#### Configuration syntax changes
+
+##### AWS
+
+The undocumented `cluster.os_name` parameter was removed since only Flatcar
+Container Linux is supported.
+
+##### Bare-metal
+
+The `cluster.os_channel` parameter got simplified by removing the `flatcar-` prefix.
+
+###### Old:
+
+```hcl
+os_channel = "flatcar-stable"
+```
+
+###### New
+
+```hcl
+os_channel = "stable"
+```
+
+##### Velero
+
+Velero requires an explicit `provider` field to select the provider.
+Example:
+
+```hcl
+component `velero` {
+  provider = "openebs"
+
+  openebs {
+    ...
+  }
+}
+```
+
+#### Cluster upgrade steps
+
+Updating multiple versions at a time is not supported so, if your cluster is older, upgrade to `v0.5.0` and only then proceed with the upgrade to `v0.6.0`.
+
+Some manual steps need to be performed when updating. In your cluster configuration directory, follow these steps:
+
+Starting from your cluster directory, export your assets directory name used in your platform configuration. It will be used in next steps:
+
+```sh
+export ASSETS_DIR="assets"
+```
+##### Only for Packet/Equinix Metal
+
+1. Disable state management for BGP sessions
+
+```bash
+cd $ASSETS_DIR/terraform
+export WORKER_POOL=""
+export WORKER_COUNT=""
+
+for ((i=0;i<$WORKER_COUNT;i++)); do
+  terraform state rm "module.worker-$WORKER_POOL.packet_bgp_session.bgp[$i]"
+done
+cd -
+```
+
+Repeat above steps for each worker pool running MetalLB with BGP enabled on it.
+
+2. Install Packet CCM
+
+  Export your Packet cluster's project ID and API Key.
+
+  ```bash
+  export PACKET_API_KEY=""
+  export PACKET_PROJECT_ID=""
+
+  echo "apiKey: $PACKET_API_KEY
+projectID: $PACKET_PROJECT_ID" > /tmp/ccm-values.yaml
+
+  # TODO: Update this to 0.6.0
+  # curl -LO https://github.com/kinvolk/lokomotive/archive/v0.5.0.tar.gz
+  # tar -xvzf v0.5.0.tar.gz
+  # helm install packet-ccm --namespace kube-system --values=/tmp/ccm-values.yaml lokomotive-0.5.0/assets/charts/control-plane/packet-ccm/
+
+  # TODO: Remove this before merging
+  curl -LO https://github.com/kinvolk/lokomotive/archive/master.zip
+  unzip master.zip
+  helm install packet-ccm --namespace kube-system --values=/tmp/ccm-values.yaml lokomotive-master/assets/charts/control-plane/packet-ccm/
+  ```
+
+3. Add permanent labels to the nodes running MetalLB, just run the following command and label will be added:
+
+Create the script which will be used to modify the env var file. So even if the node is restarted these labels will persist:
```

Comment:
- What is "the env var file"? Is it important for me as a user to know this? If so, we should explain, otherwise we might want to remove the explanation and simply tell the users which commands to run.
- Starting a sentence with "so" sounds wrong to me at the English level when the sentence continues elaborates on the previous one.
- "These labels" assumes context. We should either explain which labels we are talking about and why the user needs to worry about them, or alternatively include just a series of commands which the user needs to follow.

Reply:
Everything is moved to a separate script that will be shipped with code tar ball.